resources/repository.rb

```diff
@@ -20,19 +20,16 @@
 
 property :package_uri, String, default: lazy {
   if platform_family?('rhel')
-    'https://dl.bintray.com/rundeck/rundeck-rpm'
+    'https://packages.rundeck.com/pagerduty/rundeck/rpm_any/rpm_any/$basearch'
   else # 'debian'
-    'https://dl.bintray.com/rundeck/rundeck-deb'
+    'https://packages.rundeck.com/pagerduty/rundeck/any/'
   end
 }
-property :gpgkey, String, default: lazy {
-  if platform_family?('rhel')
-    'http://rundeck.org/keys/BUILD-GPG-KEY-Rundeck.org.key'
-  else # 'debian'
-    'https://bintray.com/user/downloadSubjectPublicKey?username=bintray'
-  end
-}
-property :gpgcheck, [true, false], default: true
+property :gpgkey, String, default: 'https://packages.rundeck.com/pagerduty/rundeck/gpgkey'
+
```

Comment:
Can you please remove these extra new lines here and below in the property declarations?